openhexa/sdk/pipelines/runtime.py

```diff
@@ -180,19 +145,27 @@ def get_pipeline_metadata(pipeline_path: Path) -> PipelineSpecs:
                         Argument("name", [ast.Constant]),
                         Argument("choices", [ast.List]),
                         Argument("help", [ast.Constant]),
-                        Argument("default", [ast.Constant]),
+                        Argument("default", [ast.Constant, ast.List]),
                         Argument("required", [ast.Constant]),
                         Argument("multiple", [ast.Constant]),
                     ),
                 )
-                try:
-                    args = param_decorator_spec["args"]
-                    inst = TYPES_BY_PYTHON_TYPE[args["type"]]()
-                    args["type"] = inst.spec_type
-
-                    pipeline.parameters.append(PipelineParameterSpecs(**args))
-                except KeyError:
-                    raise InvalidParameterError(f"Invalid parameter type {args['type']}")
+                args = param_decorator_spec["args"]
+                inst = TYPES_BY_PYTHON_TYPE[args["type"]]()
+                args["type"] = inst.expected_type
+                parameter = Parameter(
+                    code=args["code"],
+                    name=args.get("name"),
```

Comment:
I wrote my former comment on `args.get("code")` but I intended to point out all the args that are required. (So also `type`)